charts/kyverno/values.yaml

```diff
@@ -240,9 +240,9 @@ startupProbe:
     path: /health/liveness
     port: 9443
     scheme: HTTPS
-  failureThreshold: 12
-  initialDelaySeconds: 30
-  periodSeconds: 10
+  failureThreshold: 20
+  initialDelaySeconds: 2
+  periodSeconds: 6
```

Comment:
Should we update the `install.yaml` as well ?

Reply:
Good catch !

Reply:
Fixed